python/cog/predictor.py

```diff
@@ -492,7 +495,7 @@ def get_training_output_type(predictor: BasePredictor) -> Type[BaseModel]:
     train = get_train(predictor)
 
     input_types = get_type_hints(train)
-    TrainingOutputType = input_types.pop("return", None)
+    TrainingOutputType = input_types.pop("return", None)  # pylint: disable=invalid-name
```

Comment:
We should apply the same pattern as suggested in https://github.com/replicate/cog/pull/1824#discussion_r1691339203 here as well.